### PR TITLE
update the author lists

### PR DESCRIPTION
**Pull request by merlintang**

### CODE REVIEW

**Review by LianxinGao — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -94,7 +85,7 @@ Submit a pull request with a detailed explanation of your changes.
 
 ```bibtex
 @software{Ye_Multi-LoRA_2023,
-  author = {Ye, Zhengmao and Li, Dengchun and Lan, Tingfeng and Tang, Mingjie},
+  author = {Zhengmao Ye, Dengchen Li, Tingfen Lan, Yanbo Liang, Yexi Jiang, Jie Zuo, Hui Lu, Lei Duan, Mingjie Tang},
```
Comment (mikecovlee) [inline README.md:88]: Typo: replace 'dengchen' with 'dengchun' plz.

Comment (mikecovlee) [reply README.md:88]: Also 'Tingfen' to 'Tingfeng' :)

**Review by mikecovlee — CHANGES_REQUESTED** [review]

**Comment by merlintang** [comment]
fixed. @mikecovlee

**Review by mikecovlee — APPROVED** [review]
LGTM

**Pull request merged**